src/locale/__tests__/verify-translations.spec.js

```diff
@@ -1,63 +0,0 @@
-import verifyTranslations from '../tools/verify-translations';
-
-describe('verify-translations', () => {
```

Comment:
Is there any reason these tests have been deleted ?

Reply:
Ah noticed your description just right now, is there any other way to test these? Better to keep them somehow if it possible